rubocop.yml.tt

```diff
@@ -108,3 +108,7 @@ RSpec/ExampleLength:
 
 RSpec/FactoryBot/SyntaxMethods:
   Enabled: false
+
+Rails/LexicallyScopedActionFilter:
+  Exclude:
+    - "app/controllers/application_controller.rb"
```

Comment:
This rule seems useful to have everywhere except ApplicationController 

https://rails.rubystyle.guide/#lexically-scoped-action-filter